src/views/contact-your-mep.ejs

```diff
@@ -73,7 +74,7 @@
                     </figcaption>
                 </figure>
             </li>
-            <li class="mep-item p-2 w-full lg:w-1/2 bulgaria" onclick="">
+            <li class="mep-item mb-8 bulgaria" onclick="">
```

Comment:
That's what `@apply` is for, e.g.
```scss
.mep-item {
  @apply .mb-8;
}
```
Saves you from changing all elements every time ;)

Reply:
Should this be changed or is the PR fine anyway?